.travis.yml

```diff
@@ -3,6 +3,11 @@ rust:
   - stable
   - beta
   - nightly
+
+script:
```

Comment:
Thanks for the travis change! Could you update to the following, so we can use travis's build matrices for features?

```yaml
env:
  - FEATURES=""
  - FEATURES="--all-features"

script:
  - cargo test --verbose $(FEATURES)
```